e2e/pages/Payments/PaymentsPage.ts

```diff
@@ -292,5 +291,44 @@ class PaymentsPage {
     );
     expect(isPaymentPresent).toBe(true);
   }
+
+  async validateSentMessagesTab({
+    messageNotification,
+    messageContext,
+    messageType,
+    locatorNumber = 0,
+  }: {
+    messageNotification: string;
+    messageContext: string;
+    messageType: string;
+    locatorNumber?: number;
+  }) {
+    const paymentNotificationLocator = this.page
+      .locator(`:text("${messageContext} (${messageType})")`)
+      .nth(locatorNumber);
+    const messageNotificationLocator = this.page.locator(
+      `:text("${messageNotification}")`,
+    );
+    // await this.page.waitForLoadState('networkidle');
```

Comment:
In general the corrections you have provided look good to me and the test works without flakiness. 

Good job!

Two things here only:
1. Could you remove the commented line if it is not necessary?
2. Can locator be more explicitly defined or the .nth is the only solution? If it is then there is no issue :)

Reply:
1. I have removed it
2. No, because it is also be used to verify "Activity overview" on "Registration details". Then there will be multiple items with same text and also same testid.

Reply:
Approved!